src/ui/components/Toolbar.tsx

```diff
@@ -88,13 +83,21 @@ export default function Toolbar() {
   const isPaused = useSelector(selectors.hasFrames);
   const viewMode = useSelector(selectors.getViewMode);
   const selectedPrimaryPanel = useSelector(getSelectedPrimaryPanel);
-  const [hideCommentsBadge, setHideCommentsBadge] = useState(false);
+  const [showCommentsBadge, setShowCommentsBadge] = useState(false);
+  const recordingId = useGetRecordingId();
+  const { comments, loading } = hooks.useGetComments(recordingId);
+
+  useEffect(() => {
```

Comment:
:hand: It feels like more of this should be "derived state" rather than necessarily setting state in a `useEffect`, but I'd have to think about the intended behavior here more.